app/services/users.js

```diff
@@ -1,20 +1,16 @@
-const { databaseError, badRequestError } = require('../errors');
+const { databaseError } = require('../errors');
 const logger = require('../logger');
 const { user: UserModel } = require('../models');
+const { signUpSerializer } = require('../serializers/users');
+const { mapToSerializer } = require('../utils/objects');
 
 const loggerPath = 'service:users';
 
 exports.signUp = async userData => {
   try {
     const createdUser = await UserModel.create(userData);
-
-    if (!createdUser) {
-      logger.error(`${loggerPath}:signUp:logic`);
-      throw badRequestError('bad request error');
-    }
-
     logger.info(`${loggerPath}:signUp:username: ${createdUser.name}`);
-    return createdUser;
+    return mapToSerializer(createdUser, signUpSerializer);
```

Comment:
This is not bad but it seems to me that it is not the responsibility of the service to make this call, this could be done better in the controller. You could also have centralized all the logic of the serializer in the folder that you created for this.